pkg/controller/plan/util/util_suite_test.go

```diff
@@ -0,0 +1,13 @@
+package util
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+func TestVsphere(t *testing.T) {
```

Comment:
isn't this redundant here?

Reply:
nope, without it the tests won't run
but changed it to TestUtil